Makefile

```diff
@@ -507,6 +507,7 @@ $(1): $(PREFIX)/$(3)/installed/$(1)
 $(PREFIX)/$(3)/installed/$(1): $(PKG_MAKEFILES) \
                           $(PKG_PATCHES) \
                           $(PKG_TESTFILES) \
+                          $($(PKG)_FILE_DEPS) \
```

Comment:
In other parts of this definition `$(1)` is used instead of `$(PKG)`.
Can `$(PKG)` be used for all of them to support consistency and readability?